Sources/StreamChatUI/UIConfig+MessageList.swift

```diff
@@ -0,0 +1,97 @@
+        public var reactionsView: _ChatMessageReactionsView<ExtraData>.Type = _ChatMessageReactionsView<ExtraData>.self
+        public var reactionItemView: _ChatMessageReactionsView<ExtraData>.ItemView.Type =
+            _ChatMessageReactionsView<ExtraData>.ItemView.self
+    }
+
+    struct MessageContentViewSubviews {
+        public var authorAvatarView: ChatAvatarView.Type = ChatAvatarView.self
+        public var bubbleView: _ChatMessageBubbleView<ExtraData>.Type = _ChatMessageBubbleView<ExtraData>.self
+        public var metadataView: _ChatMessageMetadataView<ExtraData>.Type = _ChatMessageMetadataView<ExtraData>.self
+        public var quotedMessageBubbleView: _ChatMessageQuoteBubbleView<ExtraData>.Type = _ChatMessageQuoteBubbleView.self
+        public var attachmentSubviews = MessageAttachmentViewSubviews()
+        public var onlyVisibleForCurrentUserIndicator: ChatMessageOnlyVisibleForCurrentUserIndicator<ExtraData>.Type =
+            ChatMessageOnlyVisibleForCurrentUserIndicator<ExtraData>.self
+        public var threadArrowView: _ChatMessageThreadArrowView<ExtraData>.Type = _ChatMessageThreadArrowView<ExtraData>.self
+        public var threadInfoView: _ChatMessageThreadInfoView<ExtraData>.Type = _ChatMessageThreadInfoView<ExtraData>.self
+        public var threadParticipantAvatarView: ChatAvatarView.Type = ChatAvatarView.self
+        public var errorIndicator: _ChatMessageErrorIndicator<ExtraData>.Type = _ChatMessageErrorIndicator<ExtraData>.self
+        public var linkPreviewView: _ChatMessageLinkPreviewView<ExtraData>.Type = _ChatMessageLinkPreviewView<ExtraData>.self
+    }
+
+    struct MessageAttachmentViewSubviews {
+        public var loadingIndicator: _ChatLoadingIndicator<ExtraData>.Type = _ChatLoadingIndicator<ExtraData>.self
+        public var attachmentsView: _ChatMessageAttachmentsView<ExtraData>.Type = _ChatMessageAttachmentsView<ExtraData>.self
+        // Files
+        public var fileAttachmentListView: _ChatMessageFileAttachmentListView<ExtraData>
+            .Type = _ChatMessageFileAttachmentListView<ExtraData>.self
+        public var fileAttachmentItemView: _ChatMessageFileAttachmentListView<ExtraData>.ItemView.Type =
+            _ChatMessageFileAttachmentListView<ExtraData>.ItemView.self
+        public var fileFallbackIcon = UIImage(named: "generic", in: .streamChatUI)!
+        public var fileIcons = [AttachmentFileType: UIImage](
+            uniqueKeysWithValues: AttachmentFileType.allCases.compactMap {
+                guard let icon = UIImage(named: $0.rawValue, in: .streamChatUI) else { return nil }
+                return ($0, icon)
+            }
+        )
+        public var fileAttachmentActionIcons: [LocalAttachmentState?: UIImage] = [
+            .uploaded: UIImage(named: "uploaded", in: .streamChatUI)!,
+            .uploadingFailed: UIImage(named: "restart", in: .streamChatUI)!,
+            nil: UIImage(named: "download_and_open", in: .streamChatUI)!
```

Comment:
These images should be probably defined in `Images`

Reply:
They are already there, just didn't delete this.  Now it's done.